Aquarius/Src/Aquarius/Renderer/OrthographicCamera.cpp

```diff
@@ -0,0 +1,85 @@
+#include "OrthographicCamera.h"
```

Comment:
I think these changes are already merged to master, is it possible this branch is behind master? If so a rebase should remove these from the diff